### ISSUE TITLE
First attribute of long HTML tags is not indented

### ISSUE THREAD

**Issue by sentience** [panoply/vscode-liquid#143]
This:

```html
  <link rel="alternate" type="application/rss+xml" title="Kevin Yank posts feed" href="/feed.xml">
```

Gets formatted to this:

```html
  <link
  rel="alternate"
    type="application/rss+xml"
    title="Kevin Yank posts feed"
    href="/feed.xml">
```

That un-indented `rel` attribute looks broken.

**Comment by panoply** [comment]
Hey @sentience 

Thanks for reporting this. I believe that voids require self closing, eg:

```html
  <link
    rel="alternate"
    type="application/rss+xml"
    title="Kevin Yank posts feed"
    href="/feed.xml" />
```

At least in this version that is the case. The next release fixes this issue.

**Comment by panoply** [comment]
This was fixed in **v4.0.0** plus a who lot of different formatting issues. Thanks for submitting.
